examples/vision/oml_omniglot.py

```diff
@@ -89,31 +99,41 @@ def main(
         meta_train_accuracy = 0.0
         meta_valid_error = 0.0
         meta_valid_accuracy = 0.0
+
+        learner = maml.clone()
+        evaluation_data_all_tasks = []
+        evaluation_labels_all_tasks = []
```

Comment:
For each task we need to gather data and labels for the outer loop. Here i am using the same data used in the inner loop to calculate the validation accuracy there, since it is not the same data used for training.